front/src/app/profil-my-profil/profil-my-profil.component.ts

```diff
@@ -145,11 +144,10 @@ export class ProfilMyProfilComponent implements OnInit {
     /**
      * Description : this function keep and return current value in profil inputs
      */
-    protected getCurrentDatas()
-    {
-        this.firstName = document.getElementById('firstName').value;
-        this.civilityM = document.getElementById('inlineCheckbox1').checked;
-        this.civilityF = document.getElementById('inlineCheckbox2').checked;
+    protected getCurrentDatas() {
+        this.firstName = (<HTMLInputElement>document.getElementById('firstName')).value;
+        this.civilityM = (<HTMLInputElement>document.getElementById('inlineCheckbox1')).checked;
```

Comment:
La il faudra changer par : 
coté html :   <input class="dataForm" #checkM [checked]="civility == 'M'" type="radio" name="optradio" id="inlineCheckbox1"> ( tu met un "#unIdentifiantQueTuVeux" )
tu récupère l'élément dans le fichier .ts juste au dessus de la fonction dans laquelle tu vas utilisé la valeur récupéré:  
@ViewChild('unIdentifiantQueTuVeux') unIdentifiantQueTuVeux: ElementRef;
puis tu check si la valeur est cocher ou pas :
valeurDeLaCheckBox = this.unIdentifiantQueTuVeux.nativeElement.checked;